### PR TITLE
add

### PR DESCRIPTION
**Pull request by levdalba**

### CODE REVIEW

**Comment by levdalba** [comment]
added

File: about_lists.py [hunk 1]
```diff
@@ -1,16 +1,43 @@
 # Task 1. Create a list containing squares for number from 1 up to 10.
 # Use list comprehension
+# new_list = [x*x for x in range(10)]
```
Comment (krepysh) [inline about_lists.py]: You don't need to comment submitted code.

File: about_lists.py [hunk 2]
```diff
@@ -1,16 +1,43 @@
...
+
+# print(new_list)
+
+
+
 
 # Task 2. Create a list containig numbers leap years in the future from 2000 up to 2100
 # Use list comprehension with if condition
 # Hint: leap year is a year divisible by 4, but not by 100, unless it is divisible by 400
+# def is_leap(year):
```
Comment (krepysh) [inline about_lists.py]: same here

File: about_lists.py [hunk 3]
```diff
@@ -1,16 +1,43 @@
...
 
 
 # Task 3. You are given a dictionary with vehicles and their weights:
 vehicles = {'sedan': 1550, 'Pickup': 2000, 'bicycle': 20, 'TRUCK': 7000, 'motorcycle': 200, 'Minivan': 1700, 'SUV': 2500, 'van': 3500, 'Semi': 12000, 'Bus': 3000}
 
 # With single list comprehension select vehicle those weight is below 5000 kg
-# Convert names to upper-case within same list comprehension
\ No newline at end of file
+# Convert names to upper-case within same list comprehension
+new_list = [x.upper() for x in vehicles.keys() if vehicles[x] < 5000]
```
Comment (krepysh) [inline about_lists.py]: You can also iterate just over vehicles (without keys) it will be the same

**Review by krepysh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**